lib/cdo/i18n_string_url_tracker.rb

```diff
@@ -94,6 +90,13 @@ def log(url, source, string_key, scope = [], separator = I18n.default_separator)
     add_to_buffer(normalized_key, logged_url, source, string_key.to_s, stringified_scope, separator)
   end
 
+  def self.string_key_exists?(string_key)
+    # By default, I18n.exists? returns true if the input key is nil,
+    # but raises exception if the key is an empty string.
+    return false if string_key.nil? || string_key.empty?
+    I18n.exists? string_key, locale: I18n.default_locale
```

Comment:
I thought we couldn't use `I18n.exists?` because it doesn't use `scope`. Ask Katie about this.

Reply:
@KatieShipley what test cases should I add to the unittest in this PR to produce that issue?

Reply:
Fixed by using I18n.t when scope exists.